### PR TITLE
Integrate rbac plugin with the pre-created database generated by Backstage using TypeORM adapter.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
# What does this pull request do

Integrate rbac plugin with the pre-created database generated by Backstage using TypeORM adapter.

### CODE REVIEW

File: plugins/rh-rbac-backend/src/service/permission-policy.ts [hunk 1]
```diff
@@ -27,15 +33,31 @@ p = sub, obj, act, eft
 [policy_effect]
 e = some(where (p.eft == allow)) && !some(where (p.eft == deny))
 
+[role_definition]
+g = _, _
```
Comment (gorkem) [inline plugins/rh-rbac-backend/src/service/permission-policy.ts]: What is the reason for this change?

File: packages/backend/src/plugins/permissions.ts [hunk 2]
```diff
@@ -13,5 +16,6 @@ export default async function createPlugin(
     discovery: env.discovery,
     identity: env.identity,
     permissions: env.permissions,
+    adapterFactory: new CasbinAdapterFactory(env.config, env.database),
```
Comment (gorkem) [inline packages/backend/src/plugins/permissions.ts]: Do we need to do the creation of the AdapterFactory here? Can we do the adapter initialization inside `PolicyBuiilder.build`? This way we would not need to export the `CasbinAdapterFactory`.

File: plugins/rh-rbac-backend/src/service/permission-policy.ts (hunk 1 above)
Comment (AndrienkoAleksandr) [reply plugins/rh-rbac-backend/src/service/permission-policy.ts]: I made this change, because of the bug: https://github.com/node-casbin/typeorm-adapter/issues/62 . So I provided fix for typeORM adapter: https://github.com/node-casbin/typeorm-adapter/pull/61

File: packages/backend/src/plugins/permissions.ts (hunk 2 above)
Comment (AndrienkoAleksandr) [reply packages/backend/src/plugins/permissions.ts]: I injected adapterFactory here for testing purpose, but I can think more about that.

Comment (AndrienkoAleksandr) [reply packages/backend/src/plugins/permissions.ts]: Done. I moved adapter creation to the PolicyBuiilder.build

**Comment by AndrienkoAleksandr** [comment]
I moved these changes to the another pull request: https://github.com/janus-idp/backstage-plugins/pull/544
